Tweaks

mohawk2 opened the pull request:
This is to bring the distro up to working with, and benefiting from the features of, PDL 2.066: more integer types, and the "multi-C" feature that allows parallel-building to compile in about half the time.

It does not yet solve #1, because (as discussed on there) I do not yet understand what is wrong.

moocow-the-bovine commented on Makefile.PL:
```diff
@@ -4,7 +4,7 @@ pdlmaker_init();
 ##-- preqrequistes (for PREREQ_PM)
 my %prereq = (
-	      'PDL' => 0, #'2.4.2',
+	      'PDL' => '2.066',
```
I'd like to remain nominally compatible at least with v2.019 (stock debian buster version) for the moment.

moocow-the-bovine reviewed (commented):
@mohawk2 Thanks for the PR!  I've tweaked some of the tweaks to maintain a bit more backwards-compatibility and pushed those as https://github.com/moocow-the-bovine/PDL-CCS/tree/mohawk2-tweaks .

If you're OK with me merging that into master and/or would prefer me to push directly to your branch so I can approve & merge this PR, please let me know (I'm not sure what exactly custom dictates in this regard).

mohawk2 commented:
Thank you for your considerate approach here!

Having looked at the tweaks^2 you made, I updated my branch. Since your updates showed many of them to not be the best approach for this repo, I've dropped most of them and replaced them with yours. The only update to your updates was to change the commit messages (hope you're Ok with those, let me know if not), and to change `@$package` to `@{$package||[]}` since your initial call to the `_init` function didn't supply a `$package`.

Warning, I haven't tried this on PDL 2.019, so let me know if it breaks on there and I will fix it.
